vast/vast.cpp

```diff
@@ -140,7 +140,10 @@ int main(int argc, char** argv) {
       }
     }
     if (!plugin_found) {
-      std::cerr << "failed to find plugin: " << plugin_file << std::endl;
+      std::cerr << "failed to find plugin: " << plugin_file
```

Comment:
The message "failed to find plugin" should only be printed if the plugin really wasn't found. That message should only be added to `plugin_load_errors` only when this is really the case.

Reply:
Would you be happy with "failed to find or load plugin" instead?

Reply:
Please check back, I reworded it slightly.